ce_expansion/atomgraph/bcm.py

```diff
@@ -0,0 +1,207 @@
+from ce_expansion.data.gamma import GammaValues
+import numpy as np
+import itertools
+from ce_expansion.data.gamma import GammaValues
+import collections.abc
+import ase.units
+
+
+def recursive_update(d, u):
+    #Function found by team to update Gamma and Bulk Values
+    for k, v in u.items():
+        if isinstance(v, collections.abc.Mapping):
+            d[k] = recursive_update(d.get(k, {}), v)
+        else:
+            d[k] = v
+    return d
+
+
+class BCModel:
+    def __init__(self, atoms, bond_list, metal_types=None):
+        """
+        Based on metal_types, create ce_bulk and gamma dicts from the data given
+
+        Args:
+            atoms: ASE atoms object which contains the data of the NP being tested
+            bond_list: a CSV containing the bond data
+        KArgs:
+            metal_types: List of metals found within the nano-particle
+        """
+
+        self.atoms = atoms
+        self.bond_list = bond_list
+
+        if metal_types is None:
+            # get metal_types from atoms object
+            self.metal_types = sorted(set(atoms.symbols))
+        else:
+            # ensure metal_types to unique, sorted list of metals
+            self.metal_types = sorted(set(metal_types))
```

Comment:
I'd like to see this handle lower/uppercase inputs since it's straightforward to do. Can you change to the following:
```python
self.metal_types = sorted(set(m.title() for m in metal_types))
```